snapcraft/main.py

```diff
@@ -110,6 +113,8 @@ def main():
 
     log.configure(log_level=log_level)
 
+    common.set_enable_parallel_builds(not args['--no-parallel-build'])
+
     try:
         commands.load(cmd).main(argv=args['ARGS'])
```

Comment:
I agree with you, and I thought about that... but it honestly seemed inconsistent with how we were doing other things. I'm worried that doing that here would fragment the codebase's consistency, which may be worse than disliking how we solve an overall problem. Thoughts?